ui/app/src/components/PublishingDashboard/PublishingDashboard.tsx

```diff
@@ -70,7 +70,7 @@ export function PublishingDashboard(props: PublishingDashboardProps) {
         </Grid>
         {userPermissions.includes('get_publishing_queue') && (
           <Grid item xs={12}>
-            <PublishingQueueWidget siteId={site} />
+            <PublishingQueueWidget siteId={site} readOnly={!userPermissions.includes('publish')} />
```

Comment:
Cache the possession of the publish permission to avoid running the `.includes` twice